src/Pelagos/Bundle/AppBundle/Rabbit/Consumer/DoiConsumer.php

```diff
@@ -137,42 +137,41 @@ protected function issueDoi(Dataset $dataset, array $loggingContext)
-        }
-
-
-        $doiUtil = new DOIutil();
-
-        $generatedDOI = $doiUtil->generateDoi();
-
-        try {
-            $doiUtil->createDOI(
-                $generatedDOI,
-                $this->generateUrl('pelagos_homepage') . '/tombstone/' . $dataset->getUdi(),
-                $dataset->getAuthors(),
-                $dataset->getTitle(),
-                $dataset->getReferenceDateYear(),
-                'Harte Research Institute'
-            );
-
-            $doi = new DOI($generatedDOI);
-            $doi->setCreator($dataset->getModifier());
-            $doi->setModifier($dataset->getModifier());
-            $dataset->setDoi($doi);
+            $this->logger->info('DOI Already issued for this dataset', $loggingContext);
+        } else {
+            $doiUtil = new DOIutil();
 
-            $loggingContext['doi'] = $doi->getDoi();
-            // Log processing complete.
-            $this->logger->info('DOI Issued', $loggingContext);
-        } catch (HttpClientErrorException $exception) {
-            $this->logger->error('Error requesting DOI: ' . $exception->getMessage(), $loggingContext);
-            $issueMsg = ConsumerInterface::MSG_REJECT;
-        } catch (HttpServerErrorException $exception) {
-            $this->logger->error('Error requesting DOI: ' . $exception->getMessage(), $loggingContext);
-            //server down. wait for 10 minutes and retry.
-            sleep(self::DELAY_TIME);
-            $issueMsg = ConsumerInterface::MSG_REJECT_REQUEUE;
+            $generatedDOI = $doiUtil->generateDoi();
+
+            try {
+                $doiUtil->createDOI(
+                    $generatedDOI,
+                    $this->generateUrl('pelagos_homepage') . '/tombstone/' . $dataset->getUdi(),
+                    $dataset->getAuthors(),
+                    $dataset->getTitle(),
+                    $dataset->getReferenceDateYear(),
+                    'Harte Research Institute'
+                );
+
+                $doi = new DOI($generatedDOI);
+                $doi->setCreator($dataset->getModifier());
+                $doi->setModifier($dataset->getModifier());
+                $dataset->setDoi($doi);
+
+                $loggingContext['doi'] = $doi->getDoi();
+                // Log processing complete.
+                $this->logger->info('DOI Issued', $loggingContext);
+            } catch (HttpClientErrorException $exception) {
+                $this->logger->error('Error requesting DOI: ' . $exception->getMessage(), $loggingContext);
+                $issueMsg = ConsumerInterface::MSG_REJECT;
+            } catch (HttpServerErrorException $exception) {
+                $this->logger->error('Error requesting DOI: ' . $exception->getMessage(), $loggingContext);
+                //server down. wait for 10 minutes and retry.
+                sleep(self::DELAY_TIME);
+                $issueMsg = ConsumerInterface::MSG_REJECT_REQUEUE;
+            }
         }
-
+        
```

Comment:
extra spaces here.

Reply:
I'm unable to fix it on phpstorm, if i remove the spaces, the line gets deleted

Reply:
vim?